electron-builder.config.js

```diff
@@ -19,9 +19,10 @@ const getWindowsResources = () =>
     to: `binaries/${binary}.exe`,
   }))
 
+const stage = process.env.ITO_ENV || 'prod'
 module.exports = {
-  appId: 'ai.ito.ito',
-  productName: 'Ito',
+  appId: stage === 'prod' ? 'ai.ito.ito' : `ai.ito.ito.${stage}`,
```

Comment:
Nothing to change. I was surprised that it's a `.${stage}` and not a `-${stage}`